DEVELOPMENT.md

```diff
@@ -43,6 +44,78 @@ For example, if we want to include UI tests when we test JVM target, call this:
 ```
 Don't run any background tasks, click mouse, or press keys during the tests. Otherwise, they probably fail.
 
-#### Run samples
-- First follow the instruction here: [Building JVM bindings](#building-jvm-bindings)
-- `./gradlew :SkiaAwtSample:run`
+## Run samples
+* First follow the instruction here: [Building JVM bindings](#building-jvm-bindings)
+### samples/SkiaAwtSample
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Pskiko.version=0.0.0-SNAPSHOT
+```
+ - skiko.fps.longFrames.show=true
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Dskiko.fps.longFrames.show=true -Pskiko.version=0.0.0-SNAPSHOT
+```
+ - skiko.renderApi=SOFTWARE
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Dskiko.renderApi=SOFTWARE -Pskiko.version=0.0.0-SNAPSHOT
+```
+ - skiko.vsync.enabled=false
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Dskiko.vsync.enabled=false -Pskiko.version=0.0.0-SNAPSHOT
+```
+ - skiko.transparency=true
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Dskiko.transparency=true -Pskiko.version=0.0.0-SNAPSHOT
+```
+ - skiko.swing.interop=true
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Dskiko.swing.interop=true -Pskiko.version=0.0.0-SNAPSHOT
+```
+
+### samples/SkiaJsSample with mavenLocal
+If page in browser will be empty, try to reload it.
```

Comment:
```suggestion
If the page in the browser is empty, try to reload it.
```

Anyway, its seems like a bug, and better to not mention it in Readme? Because if the page is empty, reloading it usually is the first thing to try.

Reply:
Done

Reply:
Yeah, I think we can skip this comment. It's obviously.